### PR TITLE
Add default ToUnsafePtr

### PR DESCRIPTION
**Pull request by kzmnbrs**
This MR allows non-amd64 CPUs to use indirect.

### CODE REVIEW

**Comment by kzmnbrs** [comment]
@koykov would you kindly...

File: indirect.go [hunk 1]
```diff
@@ -1,7 +1,10 @@
+//go:build !amd64
+// +build !amd64
+
 package indirect
 
 import "unsafe"
 
 func ToUnsafePtr(ptr uintptr) unsafe.Pointer {
-	return indUP(ptr)
+	return unsafe.Pointer(ptr)
```
Comment (koykov) [inline indirect.go]: This triggers warning "Possible misuse of 'unsafe.Pointer' " and makes indirect senseless.

Provide please what kind of error you get and details of your environment.

Comment (kzmnbrs) [reply indirect.go]: I get the "it doesn't compile on ARM64 at all" kind of error:)

Comment (koykov) [reply indirect.go]: Lets try to do the following:
* rename indirect_asm64.s -> indirect_amd64.s (done)
* create file indirect_arm64.s
* put this contents to it
```
TEXT    ·indUP(SB), $0-16
PCDATA  $0, $-2
MOVD    "".ptr(FP), R0
PCDATA  $0, $-1
MOVD    R0, "".~r1+8(FP)
RET     (R30)
```

Comment (koykov) [reply indirect.go]: @kzmnbrs for further issues: I use this compiler to get asm code for different platforms: [link](https://go.godbolt.org/#g:!((g:!((h:codeEditor,i:(filename:'1',fontScale:14,fontUsePx:'0',j:1,lang:go,selection:(endColumn:2,endLineNumber:7,positionColumn:2,positionLineNumber:7,selectionStartColumn:2,selectionStartLineNumber:7,startColumn:2,startLineNumber:7),source:'//+Type+your+code+here,+or+load+an+example.%0A//+Your+function+name+should+start+with+a+capital+letter.%0Apackage+main%0A%0Aimport+(%0A++++%22unsafe%22%0A)%0A%0Afunc+indirectUnsafePtr(ptr+uintptr)+unsafe.Pointer+%7B%0A%09return+unsafe.Pointer(ptr)%0A%7D%0A%0Afunc+main()+%7B%7D'),l:'5',n:'0',o:'Go+source+%231',t:'0'),(h:compiler,i:(compiler:gl1160,deviceViewOpen:'1',filters:(b:'0',binary:'1',binaryObject:'1',commentOnly:'0',demangle:'0',directives:'0',execute:'1',intel:'0',libraryCode:'0',trim:'1'),flagsViewOpen:'1',fontScale:14,fontUsePx:'0',j:1,lang:go,libs:!(),options:'',overrides:!(),selection:(endColumn:12,endLineNumber:12,positionColumn:12,positionLineNumber:12,selectionStartColumn:12,selectionStartLineNumber:12,startColumn:12,startLineNumber:12),source:1),l:'5',n:'0',o:'+x86-64+gc+1.16+(Editor+%231)',t:'0'),(h:compiler,i:(compiler:arm64_gl116,deviceViewOpen:'1',filters:(b:'0',binary:'1',binaryObject:'1',commentOnly:'0',demangle:'0',directives:'0',execute:'1',intel:'0',libraryCode:'0',trim:'1'),flagsViewOpen:'1',fontScale:14,fontUsePx:'0',j:2,lang:go,libs:!(),options:'',overrides:!(),selection:(endColumn:22,endLineNumber:12,positionColumn:22,positionLineNumber:12,selectionStartColumn:22,selectionStartLineNumber:12,startColumn:22,startLineNumber:12),source:1),l:'5',n:'0',o:'+ARM64+gc+1.16+(Editor+%231)',t:'0')),k:100,l:'4',n:'0',o:'',s:2,t:'0')),version:4)

Perhaps, avo is better solution, but I don't have experience wit it.

**Pull request merged**